config/datasets/ot_gcp.yaml

```diff
@@ -40,19 +40,24 @@ target_index: ${datasets.release_folder}/targets # OTP 23.12 data
 
 gene_interactions: ${datasets.release_folder}/interaction # OTP 23.12 data
 eqtl_catalogue_paths_imported: ${datasets.inputs}/preprocess/eqtl_catalogue/tabix_ftp_paths_imported.tsv
+finngen_finemapping_results_url: ${datasets.inputs}/Finngen_susie_finemapping_r10/full
```

Comment:
I would avoid using `url` and use `path` or something similar instead. `url` refers to a location in the web which is not the case.